src/Global/organisms/MainMenu.tsx

```diff
@@ -46,7 +46,38 @@ const SVGObjectText = styled.object`
   max-width: 180px;
 `;
 
-const NavigationContainer = styled.div``;
+const GradientContainer = styled.div`
+  &::after {
+    content: '';
+    background: radial-gradient(farthest-side at 50% 100%,rgba(246, 247, 249, 0) 1%, #F6F7F9 120%) center bottom; //Shodow bottom
```

Comment:
```suggestion
    background: linear-gradient(to bottom, transparent, #e5e5e5); //Shodow bottom
```
Just a suggestion,
Because current one is not even seem like menu items are behind it. You can also use a different one if that looks better.

**Before**

![image](https://user-images.githubusercontent.com/95204740/227990421-adb6e6c4-1c85-47bb-baae-2b4f9e9f52c8.png)

**After**

![image](https://user-images.githubusercontent.com/95204740/227990351-28a633f7-6b62-4ebc-a7ce-65d0cdb77a6c.png)